apache/zeppelin
# [ZEPPELIN-3848] zeppelin-web dev mode does not work

### What is this PR for?
zeppelin-web development mode is broken. `npm run dev` throws error

```
Error: Cannot find module 'webpack/bin/config-yargs'
```

This PR fixes error by upgrading webpack from version 1.x to 4.x


### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-3848

### How should this be tested?
check if `npm run dev` works under zeppelin-web dir. And able to browse localhost:9000

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no
